cicd/gitlab/parts/python.gitlab-ci.yml

```diff
@@ -116,12 +116,15 @@ python:coverage:cobertura:
   extends: .python:coverage
   script:
     - coverage xml -o $RUN_DIR/cobertura_coverage.xml
+    - genbadge coverage -i $RUN_DIR/cobertura_coverage.xml -o $RUN_DIR/coverage-badge.svg
```

Comment:
do we want to have coverages stages separated?
can we use `-o $RUN_DIR/coverage/coverage-badge.svg` so we won't need to `mv` anything in `pages` job?

Reply:
I have just realised `cobertura` is only useful for merge requests (GitLab), so it is useless for GitHub PRs. Will remove it and merge the coverage stages :+1: